From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sun, 10 Jun 2018 16:46:57 +0200
Subject: [PATCH] {BUG in 4.18-rc1} drivers/media/platform/davinci: do not use
 uninitialized value

Variable frame_size is not used in isif_config_ycbcr
---
 drivers/media/platform/davinci/isif.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/platform/davinci/isif.c b/drivers/media/platform/davinci/isif.c
index 47cecd10eb9f..c283c89e5ffd 100644
--- a/drivers/media/platform/davinci/isif.c
+++ b/drivers/media/platform/davinci/isif.c
@@ -884,7 +884,7 @@ static int isif_set_hw_if_params(struct vpfe_hw_if_param *params)
 static int isif_config_ycbcr(void)
 {
 	struct isif_ycbcr_config *params = &isif_cfg.ycbcr;
-	struct vpss_pg_frame_size frame_size;
+	/*struct vpss_pg_frame_size frame_size;*/
 	u32 modeset = 0, ccdcfg = 0;
 	struct vpss_sync_pol sync;
 
@@ -979,7 +979,7 @@ static int isif_config_ycbcr(void)
 		sync.ccdpg_hdpol = params->hd_pol;
 		sync.ccdpg_vdpol = params->vd_pol;
 		dm365_vpss_set_sync_pol(sync);
-		dm365_vpss_set_pg_frame_size(frame_size);
+		/*dm365_vpss_set_pg_frame_size(frame_size);*/
 	}
 	return 0;
 }
-- 
